src/scripts/cronjobs/updateBreachesInRemoteSettings.ts

```diff
@@ -33,19 +33,36 @@
  *
  */
 
-import AppConstants from "../../appConstants";
 import * as HIBP from "../../utils/hibp";
 
 type RemoteSettingsBreach = Pick<
   HIBP.HibpGetBreachesResponse[number],
   "Name" | "Domain" | "BreachDate" | "PwnCount" | "AddedDate" | "DataClasses"
 >;
 
+const FX_REMOTE_SETTINGS_WRITER_USER =
+  process.env.FX_REMOTE_SETTINGS_WRITER_USER;
+const FX_REMOTE_SETTINGS_WRITER_PASS =
+  process.env.FX_REMOTE_SETTINGS_WRITER_PASS;
+const FX_REMOTE_SETTINGS_WRITER_SERVER =
+  process.env.FX_REMOTE_SETTINGS_WRITER_SERVER;
+
+if (
+  !FX_REMOTE_SETTINGS_WRITER_USER ||
+  !FX_REMOTE_SETTINGS_WRITER_PASS ||
+  !FX_REMOTE_SETTINGS_WRITER_SERVER
+) {
```

Comment:
Seems like you could use `getEnvVarsOrThrow` and catch it + `process.exit` (I think throwing from here would the process to exit anyway but being explicit is probably better just in case it's refactored later)

Reply:
Yeah, I was actually considering that too, but then I figured I'd have to wrap the entire file in a `try {}` block, which... Isn't that big of a deal, but felt a little bit icky, so I stuck with mostly matching the existing code. I hadn't realised yet that throwing would cause an exit anyway, so that could actually be a good compromise too. What do you think?